buildSrc/src/main/resources/forbidden/jdk-signatures.txt

```diff
@@ -88,7 +88,6 @@ java.lang.Thread#getAllStackTraces()
 
 @defaultMessage Stopping threads explicitly leads to inconsistent states. Use interrupt() instead.
 java.lang.Thread#stop()
-java.lang.Thread#stop(java.lang.Throwable)
```

Comment:
Why is this removed from forbidden apis? The forbidden signatures should against our minimum runtime, which is still java 8.

Reply:
The method was removed in jdk 11 and causing the test to fail. 
Do we have per JDK version forbidden APIs ?

Reply:
We do not. I think this is related to running forbidden apis in a separate jvm. We should always be running it with the minimum supported version.

Reply:
Can you please comment this out and reference the issue to run forbidden apis in a forked jvm.

Reply:
If this issue was the only reason to no longer use the gradle plugin and fork instead, I can just say: The above forbidden-api signature can be removed anyways from elasticsearch (together with the previous one). Forbiddenapis has those signatures already in its "jdk-deprecated" ones, as it's a known problem - so there is no need to have it in Elasticsearch's custom signatures. "jdk-deprecated" automatically handles deprecations that were removed.

IMHO, #31715 was not a good idea, sorry!

Reply:
https://github.com/policeman-tools/forbidden-apis/blob/master/src/main/resources/de/thetaphi/forbiddenapis/signatures/jdk-deprecated-1.8.txt#L175-L180

and

https://github.com/policeman-tools/forbidden-apis/blob/master/src/main/resources/de/thetaphi/forbiddenapis/signatures/jdk-deprecated-1.8.txt#L7